DEPS

```diff
@@ -4264,7 +4264,7 @@ hooks = [
     'action': ['python3', 'src/build/util/lastchange.py',
                '-o', 'src/build/util/LASTCHANGE',
                # Cobalt addition, don't look for Change-Id in commits.
-               '--filter=^'
+               '--filter=\(#[0-9]{4,}\)'
```

Comment:
This seems to advanced for the DEPS parser. How about `\(#[0-9]+\)`? Is there a specific case you are trying to guard against?

Reply:
changed